docs/developer-docs/daos/sns/managing/making-proposals.mdx

```diff
@@ -601,6 +601,62 @@ quill sns make-proposal <PROPOSER_NEURON_ID> --proposal '(
 quill send message.json
 ```
 
+### `ManageLedgerParameters`
+The proposal `ManageLedgerParameters` can be used to update some of the SNS ledger canister's parameters.
+For now, the only ledger parameter that can be changed is the transfer fee.
+Later, additional paramters might be added.
+Fields where a value is set to `None` will remain unchanged.
+
+#### Relevant type signatures
+
+```candid
+  type ManageLedgerParameters = record {
+    transfer_fee : opt nat64
+  };
+```
+
+Example in bash:
+
+```bash
+TODO
```

Comment:
@daniel-wong-dfinity-org could you please fill this in?

Reply:
quill sns make-proposal <PROPOSER_NEURON_ID> --proposal '(                                  
    record {                                             
        title = "Change Our Ledger Fee to 314_159 e8s";
        url = "https://sns-examples.com/proposal/42";
        summary = "Change Our Ledger Fee to 314_159 e8s.";
        action = opt variant {
            ManageLedgerParameters = record {
                transfer_fee = opt 314_159;
            }                                    
        }
    }
)' --canister-ids-file openchat_canister_ids.json > message.json

quill send message.json